moveit_core/CMakeLists.txt

```diff
@@ -46,15 +42,21 @@ else()
   message(STATUS "Version of Bullet too old or not available: disabling Bullet collision detection plugin. Try using Ubuntu 18.04 or later.")
 endif()
 
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(LIBFCL_PC REQUIRED fcl)
-set(LIBFCL_INCLUDE_DIRS ${LIBFCL_PC_INCLUDE_DIRS})
-# find *absolute* paths to LIBFCL_LIBRARIES
-set(LIBFCL_LIBRARIES)
-foreach(_lib ${LIBFCL_PC_LIBRARIES})
-  find_library(_lib_${_lib} ${_lib} HINTS ${LIBFCL_PC_LIBRARY_DIRS})
-  list(APPEND LIBFCL_LIBRARIES ${_lib_${_lib}})
-endforeach()
+find_package(fcl QUIET)
+if (fcl_FOUND)
+  set(LIBFCL_LIBRARIES ${FCL_LIBRARIES})
+  set(LIBFCL_INCLUDE_DIRS ${FCL_INCLUDE_DIRS})
+else()
```

Comment:
Can you add a comment why the old-style pkgconfig way is still required?

Reply:
@wolfv - you played with this; could you please address this comment?

Reply:
Because the dcl from Ubuntu doesn't support FCL CMake properly IIRC.